package.json

```diff
@@ -4,9 +4,9 @@
   "description": "Wraps libsodiums secretstream in a higher level abstraction",
   "main": "index.js",
   "dependencies": {
+    "b4a": "^1.1.1",
     "sodium-universal": "^3.0.4"
   },
-  "devDependencies": {},
```

Comment:
keep this, i like to have the empty map there so npm adds the deps in the same place (we should prob add tests also haha)

Reply:
It was npm itself that nuked it so it might be a bit of an uphill battle 🙈

Reply:
Can you just add standard for now?